web_src/js/webcomponents/overflow-menu.js

```diff
@@ -22,18 +23,34 @@ window.customElements.define('overflow-menu', class extends HTMLElement {
               items[0].focus();
             }
           }
-        } else if (e.code === 'Escape') {
+        } else if (e.key === 'Escape') {
           e.preventDefault();
+          e.stopPropagation();
           this.button._tippy.hide();
           this.button.focus();
-        } else if (e.code === 'Space') {
-          // eslint-disable-next-line unicorn/no-lonely-if
-          if (document.activeElement.matches('[role="menuitem"]')) {
-            e.preventDefault();
+        } else if (e.key === ' ' || e.code === 'Enter') {
+          e.preventDefault();
+          e.stopPropagation();
+          if (document.activeElement?.matches('[role="menuitem"]')) {
             document.activeElement.click();
           }
+        } else if (e.key === 'ArrowDown') {
+          e.preventDefault();
+          e.stopPropagation();
+          if (document.activeElement?.matches('.tippy-target')) {
+            document.activeElement.querySelector('[role="menuitem"]').focus();
+          } else if (document.activeElement?.matches('[role="menuitem"]')) {
+            document.activeElement.nextElementSibling?.focus();
+          }
+        } else if (e.key === 'ArrowUp') {
+          e.preventDefault();
+          e.stopPropagation();
```

Comment:
The same as above, it should only "prevent" if and only if the event is really handled by us.

Reply:
fixed with https://github.com/go-gitea/gitea/pull/29400/commits/97ed9ed33f871d36c5885f50ccc868d5fe6383bf